src/elements/content-sidebar/__tests__/BoxAISidebar.test.tsx

```diff
@@ -186,6 +187,24 @@ describe('elements/content-sidebar/BoxAISidebar', () => {
         });
     });
 
+    test('should call setHasQuestions on load if questions is not empty', async () => {
```

Comment:
should we have a test that verifies that setHasQuestions is not called if the questions are not empty?

Reply:
setHasQuestions should be called whether questions are empty or not 
should be called with "true" when questions are not empty
should be called with "false" when questions are empty
will update test for these two different case